sweddit_combined/history/countries/JIN - Jin.txt

```diff
@@ -1,6 +1,37 @@
 government = monarchy
-add_government_reform = chinese_warlord
+add_government_reform = chinese_hegemon_reform
 technology_group = chinese
 religion = confucianism
 primary_culture = jin
+add_accepted_culture = chihan
 capital = 693	#Taiyuan
+1434.06.15 = { #Coronation
+	monarch = {
+		name = Wengong
+		dynasty = Jin
+		adm = 6
+		dip = 4
+		mil = 5
+		birth_date = 1377.04.25
+		female = no
+		regent = no
+		religion = confucianism
+		culture = jin
+	}
+	heir = {
+		name = Xiang
+		monarch_name = Xiang
+		dynasty = Jin
+		birth_date = 1404.10.17
+		death_date = 1445.01.01
+		claim = 95
+		adm = 3
+		dip = 3
+		mil = 4
+	}
+	clear_scripted_personalities = yes
+	add_ruler_personality = martial_educator_personality
+	add_ruler_personality = lawgiver_personality
+	add_ruler_personality = well_connected_personality
+	add_heir_personality = tactical_genius_personality
```

Comment:
Will all of these go to the monarch and none to the ruler?

And same as above, should these really be set.